src/PdfReader/index.tsx

```diff
@@ -329,25 +331,53 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
     });
   }, [state.scale]);
 
+  // If this is a single PDF webpub, goToPage navigates to the page using the destination array.
+  // If this is a multi PDF webpub, goToPage loads the new resource.
   const goToPage = React.useCallback(
-    async (href) => {
-      const getIndexFromHref = (href: string): number => {
-        const index = manifest?.readingOrder?.findIndex((link) => {
-          return link.href === href;
+    (target: string | any[]) => {
```

Comment:
This works for now, but I think in the future we should try to make this work with a Readium `Locator` object, which I believe is supposed to handle cases where it is a page or specific location within a resource. Thus we can avoid having the react-pdf specific logic and types leak out of `usePdfReader`

Reply:
[Here is locator reference](https://readium.org/architecture/models/locators/)

Reply:
This is what a pdf locator would look like:
```
{
  "href": "http://example.com/document",
  "type": "application/pdf",
  "title": "Page 5",
  "locations": {
    "fragments": ["page=5", "viewrect=50,50,640,480"],
    "progression": 0.12703,
    "totalProgression": 0.12703
  }
}
```